backstop/tests/file-upload.html

```diff
@@ -91,28 +138,16 @@ <h2>Table</h2>
                   aria-label="Apply filter"
                   type="button"
                 >
-                  <!-- Use filter-hollow.svg from @itwin/itwinui-icons package. -->
-                  <svg
+                  <svg-filter-hollow
                     class="iui-icon"
                     aria-hidden="true"
-                    viewBox="0 0 16 16"
-                  >
-                    <path
-                      d="m15 1v.5l-5.6 4.7-.4.3v.5 5.5l-2 1.5v-7-.5l-.4-.3-5.6-4.7v-.5zm1-1h-16v2l6 5v9l4-3v-6l6-5z" />
-                  </svg>
+                  ></svg-filter-hollow>
                 </button>
                 <div class="iui-cell-end-icon">
-                  <!-- Use sort-down.svg from @itwin/itwinui-icons package. -->
-                  <svg
+                  <svg-sort-down
                     class="iui-icon iui-sort"
-                    viewBox="0 0 16 16"
                     aria-hidden="true"
-                  >
-                    <path d="m7 0v12.7l-3.8-3.7-1.2 1.2 6 5.8 1.2-1.2 4.8-4.6-1.2-1.2-3.8 3.7v-12.7z" />
-                  </svg>
-                </div>
-                <div class="iui-resizer">
```

Comment:
why this dissapeared?

Reply:
Jon and I noticed that the resizer was showing above the file drag background (because it has z-index: 1), so removed it 😄

Reply:
and now what? never come back?